src/romaine/core.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+
+class Core():
+    """
+        The core of the Romaine, provides BDD test API.
+    """
+    # All located features
+    features = []
+
+    def locate_features(self, path):
+        """
+            Locate any features given a path.
+
+            Keyword arguments:
+            path -- The path to search for features, recursively.
+
+            Returns:
+            List of features located in the path.
+        """
+        walked_paths = os.walk(path)
+
+        feature_candidates = []
+
+        for walked_path in walked_paths:
```

Comment:
I think we should expand `walked_path` as `(base_dir, subdirs, files)` - this would make the below comments unnecessary, and improve naming on the below code.